### PR TITLE
Enable JARM by default and disable on request

### PR DESCRIPTION
**Pull request by Shenali-SJ**
### Purpose
This enables JARM by default and also allows disabling the feature on request.

### Related Issues
- https://github.com/wso2/product-is/issues/16618

### Followup Actions
- Bump the framework version
- Update the documentation

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json [hunk 1]
```diff
@@ -217,6 +217,17 @@
 
   "oauth.global_rbac_scope_issuer.enable": false,
 
+  "oauth.jarm.enable": true,
```
Comment (chamathns) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:220]: do we need a separate one here?

**Review by chamathns — COMMENTED** [review]
let's add a reference to the identity.xml file as well. To denote how the final configurations will look like.
for reference: https://github.com/wso2/carbon-identity-framework/pull/2968

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json (hunk 1 above)
Comment (Shenali-SJ) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:220]: This is resolved with the commit ID c6da8c20e46455e080693db31c128ff6839d994d

**Review by chamathns — APPROVED** [review]

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json [hunk 2]
```diff
@@ -217,7 +217,6 @@
 
   "oauth.global_rbac_scope_issuer.enable": false,
 
-  "oauth.jarm.enable": true,
```
[the remark this answers is not in the document]
Comment (Shenali-SJ) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json]: This is updated with the commit da13cb56095368be809ebb6e23c21ca2eb1a2299

**Review by chamathns — APPROVED** [review]

**Pull request merged**